cocos/2d/assembler/label/text-processing.ts

```diff
@@ -573,6 +607,10 @@ export class TextProcessing {
     }
 
     private _setupOutline (style: TextStyle): void {
+        this._context!.shadowBlur = 0;
+        this._context!.shadowOffsetX = 0;
+        this._context!.shadowOffsetY = 0;
+        this._context!.miterLimit = 2;
```

Comment:
Why do we need to set miterLimit to 2?

Reply:
Hoping for a clearer stroke, no effect，i will remove